tests/spread/integration/perl/task.yaml

```diff
@@ -0,0 +1,7 @@
+summary: Integration tests for perl
+
+execute: |
+  rootfs=$(install-slices base-files_base bash_bins coreutils_bins perl_bins)
+  mkdir -p "$rootfs/dev"
+  touch "$rootfs/dev/null"
```

Comment:
Are we using the `/dev/null` inside the rootfs?

Reply:
Yes, `perl` is using it.

Reply:
Question for myself, why do we need to touch `dev/null`? Shouldn't the container runtime mount the devfs?

Reply:
Yes and no. In the current implementation, the spread test runs in an unprivileged container. However, our test uses `chroot` within that container to run the smoke test. As a result, we won't have `/dev` in the `chroot`'ed environment.